gis/geo_schelling/README.md

```diff
@@ -16,10 +16,10 @@ NUTS-2 regions are the GeoAgents. The neighbors of a polygon are considered thos
 
 ## How to Run
 
-To run the model interactively, run `mesa runserver` in this directory. e.g.
+To run the model interactively, run `solara run app.py` in this directory. e.g.
 
 ```bash
-mesa runserver
+solara run app.py
 ```
 
-Then open your browser to [http://127.0.0.1:8521/](http://127.0.0.1:8521/) and press `Start`.
+Then open your browser to [http://127.0.0.1:8765/](http://127.0.0.1:8765/) and press `Start`.
```

Comment:
There is no "Start" button anymore. It is "▶".

Reply:
You're right! Fixed now.